solr/core/src/test/org/apache/solr/security/JWTAuthPluginTest.java

```diff
@@ -44,7 +44,6 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
-import org.mortbay.util.ajax.JSON;
```

Comment:
Avoid using old mortbay utilities for converting JSON. Uses existing Solr Utils to convert from JSON string.